File: Exercicios_guanaraba/Com_Ruby/exercicios/ex015_Aluguel_de_carros.rb
=begin
Escreva um programa que pergunte a quantidade de KM percorridos por um carro alugado 
e a quantidade de dias pelos quais ele foi alugado. Calcule o preço a pagar, 
sabendo que o carro custa R$60 por dia e R$0,15 por km rodado.
=end


puts 'Quantos dias você ficou com o carro?:'
dias = gets.chomp.to_i

puts 'Quantos KM você utilizou?: '
KM = gets.chomp.to_f

pago = (dias*60)+ (KM*0.15)
puts "A diaria de #{dias} dias, custou R$#{dias*60}"
puts "Os #{KM}Km utilizados, custou R$#{KM*0.15}"
puts "No total ficou #{pago}"

File: Exercicios_guanaraba/Com_Dart/bin/ex015_Aluguel_de_Carros.dart
/*
Escreva um programa que pergunte a quantidade de KM percorridos por um carro alugado
e a quantidade de dias pelos quais ele foi alugado. Calcule o preço a pagar, sabendo
que o carro custa R$60 por dia e R$0,15 por km rodado.
*/

import 'dart:io';

void main(){

  print ('Quantos dias você ficou com o carro?: ');
  double dias = double.parse(stdin.readLineSync());
  
  print ('Quantos KM você utilizou?: ');
  double km = double.parse(stdin.readLineSync());

  double pago = (dias*60) + (km*0.15);

  print ('A diaria de $dias dias, custou R\$' + (dias*60).toString());

  print ('Os $km\Km utilizados, custou R\$' + (km*0.15).toString());

  print ('No total ficou: $pago');

}